google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/TableAdminExample.java

```diff
@@ -0,0 +1,373 @@
+        GCRules.GCRULES.intersection().rule(maxAgeRule3).rule(versionRule3);
+
+    // Creates column family with given GC rule.
+    try {
+      ModifyColumnFamiliesRequest columnFamiliesRequest4 =
+          ModifyColumnFamiliesRequest.of(tableId).addFamily(COLUMN_FAMILY_4, intersectionRule1);
+      adminClient.modifyFamilies(columnFamiliesRequest4);
+      System.out.println("Created column family: " + COLUMN_FAMILY_4);
+    } catch (AlreadyExistsException e) {
+      System.err.println(
+          "Failed to create column family with rule, already exists: " + e.getMessage());
+    }
+    // [END bigtable_create_family_gc_intersection]
+  }
+
+  /** Demonstrates how to create a nested rule using the IntersectionRule and UnionRule. */
+  public void nestedRule() {
+    System.out.printf("%nCreating column family %s with a nested GC rule%n", COLUMN_FAMILY_5);
+    // [START bigtable_create_family_gc_nested]
+    // Creates a nested GC rule:
+    // Drop cells that are either older than the 10 recent versions
+    // OR
+    // Drop cells that are older than a month AND older than the 2 recent versions
+    GCRules.VersionRule versionRule4 = GCRules.GCRULES.maxVersions(10);
+    GCRules.DurationRule maxAgeRule4 = GCRules.GCRULES.maxAge(30, TimeUnit.DAYS);
+    GCRules.VersionRule versionRule5 = GCRules.GCRULES.maxVersions(2);
+    GCRules.IntersectionRule intersectionRule2 =
+        GCRules.GCRULES.intersection().rule(maxAgeRule4).rule(versionRule5);
+    GCRules.UnionRule unionRule2 =
+        GCRules.GCRULES.union().rule(intersectionRule2).rule(versionRule4);
+
+    // Creates column family with given GC rule.
+    try {
+      ModifyColumnFamiliesRequest columnFamiliesRequest5 =
+          ModifyColumnFamiliesRequest.of(tableId).addFamily(COLUMN_FAMILY_5, unionRule2);
+      adminClient.modifyFamilies(columnFamiliesRequest5);
+      System.out.println("Created column family: " + COLUMN_FAMILY_5);
+    } catch (AlreadyExistsException e) {
+      System.err.println(
+          "Failed to create column family with rule, already exists: " + e.getMessage());
+    }
+    // [END bigtable_create_family_gc_nested]
+  }
+
+  /** Demonstrates how to list a table's column families. */
+  public void listColumnFamilies() {
+    System.out.println("\nPrinting ID and GC Rule for all column families");
+    // [START bigtable_list_column_families]
+    // Lists all families in the table with GC rules.
+    try {
+      Table table = adminClient.getTable(tableId);
+      Collection<ColumnFamily> columnFamilies = table.getColumnFamilies();
+      for (ColumnFamily columnFamily : columnFamilies) {
+        printColumnFamily(columnFamily);
+      }
+    } catch (NotFoundException e) {
+      System.err.println(
+          "Failed to list column families from a non-existent table: " + e.getMessage());
+    }
+    // [END bigtable_list_column_families]
+  }
+
+  /** Demonstrates how to modify a column family's rule. */
+  public void modifyColumnFamilyRule() {
```

Comment:
How is this different then the examples above?

Reply:
This method is modifying a columnFamily's rule, as opposed to the above methods are actually adding a columnFamily with a rule to the table. 
Above it does: ModifyColumnFamiliesRequest.of(tableId).**addFamily**(COLUMN_FAMILY, rule). 
Here is does: ModifyColumnFamiliesRequest.of(tableId).**updateFamily**(COLUMN_FAMILY, rule)